dags/surveymonkey_data_import_pipeline.py

```diff
@@ -117,6 +119,39 @@ def surveymonkey_survey_questions_etl(**kwargs):
         )
 
 
+def surveymonkey_survey_answers_etl(**kwargs):
+    data_config = data_config_from_xcom(kwargs)
+    survey_id_list_from_df = get_distinct_values_from_bq(
+        project_name=data_config.project_name,
+        dataset_name=data_config.dataset_name,
+        table_name_source=data_config.survey_list_table_name,
+        column_name=data_config.survey_id_column_name
+    ).values.tolist()
+
+    for survey_id_list in survey_id_list_from_df:
+        survey_id = str(survey_id_list[1])
+        LOGGER.info("answers for survey_id: %s", survey_id)
+        list_of_answers_of_one_survey = get_survey_answer_details(
+            access_token=get_surveymonkey_access_token(),
+            survey_id=survey_id
+        )
+        len_of_answer_list = len(list_of_answers_of_one_survey)
+        LOGGER.info("length of answers is %s", len_of_answer_list)
+        if len_of_answer_list > 0:
+            for answers_dict in list_of_answers_of_one_survey:
+                survey_answers_list = [
+                    get_bq_json_for_survey_answers_response_json(answers_dict)
+                ]
+                load_given_json_list_data_from_tempdir_to_bq(
```

Comment:
comment; perhaps it would be worth considering loading answers of all surveys in one go?

Reply:
actually looking at this again, this is related to a single survey only, isn't it?
In this case it definitely seems to be worth to load all answers of a survey in one go.

Reply:
Yes it is related to just one survey, we can append all answers in same list before loading but in that case it can be a huge list.

Reply:
Some thoughts on potential benefits:

- It will be semi-atomic: Either we imported the answers of a survey or we haven't (there is no chance of just having imported the first five answers).
- It should be a lot more efficient
- It may even help with schema detection as it can look at more answers

But maybe I don't fully appreciate how huge the list can get. What sort of size are we talking about?

Reply:
Actually size of the list will be updated after a meeting with Mael, maybe we will eliminate some of the surveys to get their data. I will change it as a one-list to load.

Reply:
How will the size of the list change after the meeting?
In any case I think I got confused with the answer response as well.
Importing the responses probably doesn't need to be atomic, because in "normal" operation we may start importing data before the survey is closed.
But calling the bigquery import for every response individually seem to be too frequent still (e.g. 2000 times).
So at least batching could be useful.